### DISCUSSION TITLE
[Please read the message below] I have not seen an example of working with an external server

### DISCUSSION THREAD

**Discussion by sh1man** [solidjs/solid-start#1308] [Q&A]
I love working with python, so I chose fastapi (backend server) as my server. Does it make sense to use solidstart then? In this case, is it better to use solid js?

**Comment by ryansolid** [comment]
It depends what your goal is by backend. If you mean you want to use Python to render web pages then I'd probably go with just SolidJS I'm not sure Start offers much here. SolidStart assumes that the backend runs a bit of javascript (or is static) for serving pages. Truthfully, you could make a site using Start with `ssr: false` but it will render a static `index.html` page and require no JS on the server. 

If you mean a backend that serves data and APIs then Start could be used a separate JS web server in tandem.

**Reply by sh1man999** [reply]
Yes, a used backend that serves data and APIs
